tests/test_cases/test_discovery/test_discovery.py

```diff
@@ -470,3 +470,34 @@ def type_check_verilog(dut):
         tlog.info("Handle %s" %  (handle[0]._fullname,))
         if handle[0]._type != handle[1]:
             raise TestFailure("Expected %s found %s for %s" % (handle[1], handle[0]._type, handle[0]._fullname))
+
+
+# identifiers starting with `_` are illegal in VHDL
+@cocotb.test(skip=cocotb.LANGUAGE in ["vhdl"],
+             expect_error=AttributeError)
```

Comment:
Better to do a single try/except within the other test.

Reply:
Or rather, use https://github.com/cocotb/cocotb/blob/7db4c187e2c303d31f8226782b2afece6432eade/tests/test_cases/test_cocotb/common.py#L48

Reply:
Done, did you mean like that?

Reply:
Yes!